agent/containers/images/all/Makefile

```diff
@@ -0,0 +1,11 @@
+distros = centos-8 centos-7 fedora-32 fedora-31
+
+all: ${distros}
+
+${distros}: clean
+	jinja2 Dockerfile.j2 -D distro=$@ > ./Dockerfile
```

Comment:
Having never used jinja2 before, I tried to get one of these makes to actually run. I installed (default) jinja2 and jinja2-cli in a pipenv; the make fails when jinja2cli throws InvalidDataFormat: yml.

I'm going to assume I've got the wrong setup; any hints?

Reply:
FWIW, I have been running this code on Fedora 31, using the `python3-jinja2` and `python3-jinja2-cli` RPMs.